src/main/scala/mesosphere/marathon/core/launcher/impl/OfferProcessorImpl.scala

```diff
@@ -81,10 +79,9 @@ private[launcher] class OfferProcessorImpl(
   }
 
   private def logOffer(offer: Offer): Unit = {
-    val raml = offerWrites(offer)
-    val json = Json.toJson(raml)
-    val compact = json.toString()
-    logger.info(s"Processing offer: $compact")
+    val offerId = offer.getId.getValue
+    val agentId = offer.getSlaveId.getValue
+    logger.info(s"Processing offer: offerId $offerId, agentId $agentId")
```

Comment:
Why this change? Why not printing entire offers?

Also, could you please log and warn before doing anything with an offer? Meaning this
```
     incomingOffersMeter.increment()
     offerStreamInput.offer(offer)
     warnOnZeroResource(offer)
     logOffer(offer)
```

should be replaced with

```
     incomingOffersMeter.increment()
     logOffer(offer)
     warnOnZeroResource(offer)
     offerStreamInput.offer(offer)
```

Reply:
Why offer summary? See this part of the discussion on the related issue:

![screen shot 2018-07-05 at 2 16 38 pm](https://user-images.githubusercontent.com/183/42345905-1733769e-805e-11e8-8da1-1829b9bdf39f.png)

Perhaps if debug logging is on we can print the whole offer. I think it is excessive for INFO level.

Reply:
I think it would be excessive if we were to print offers on multiple lines. In the screenshot you posted, I support printing them as one-liners compactly formatted. This is what you referred to as a brief summary as well.